src/app/app.config.ts

```diff
@@ -0,0 +1,8 @@
+import { ApplicationConfig } from '@angular/core';
```

Comment:
suggestion

          import { ApplicationConfig } from '@angular/core'; // Importing ApplicationConfig from Angular core

          Explantion: It's good practice to add a comment for important imports for clarity.